drivers/storage/portworx/portworx_test.go

```diff
@@ -1075,6 +1078,26 @@ func TestStorageClusterDefaultsForCSI(t *testing.T) {
 	require.Empty(t, cluster.Status.DesiredImages.CSINodeDriverRegistrar)
 	require.Empty(t, cluster.Status.DesiredImages.CSIResizer)
 	require.Empty(t, cluster.Status.DesiredImages.CSISnapshotter)
+
+	// If CSI feature flag is true and CSI Spec is false, honor feature flag true
+	cluster.Spec.CSI.Enabled = false
```

Comment:
Can we add a simple test with CSI spec as empty and feature gate is set to true value? This will test the most common scenario where existing customers are moved to CSI spec.